substrate/frame/referenda/src/lib.rs

```diff
@@ -230,10 +238,13 @@ pub mod pallet {
 		/// Information concerning the different referendum tracks.
 		#[pallet::constant]
 		type Tracks: Get<
```

Comment:
(as a side note: this `Get` doesn't seem really needed, we could have add it in the constant using `pallet::extra_constant` attribute.)

Reply:
Done. Great suggestion.